### PR TITLE
nothing big

### PR DESCRIPTION
**Pull request by Bikaze**
good push

### CODE REVIEW

**Review by Bikazemcclent — CHANGES_REQUESTED** [review]
nothing really big

**Comment by Bikaze** [comment]
ok

**Review by Bikazemcclent — COMMENTED** [review]
This should be improved

File: collaboration/README.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Collaboration
 
-<!-- group norms summary -->
```
Comment (Bikazemcclent) [inline collaboration/README.md:3]: what's this?

**Review by Bikazemcclent — COMMENTED** [review]
Ok look at that.

File: collaboration/README.md (hunk 1 above)
Comment (Bikazemcclent) [reply collaboration/README.md:3]: This is actually a comment.

Comment (Bikaze) [reply collaboration/README.md:3]: Yes don't you see that it is a comment?

Comment (Bikazemcclent) [reply collaboration/README.md:3]: Ok I see

File: collaboration/README.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+## Group Norms
+
+- We will respect each other's time while remaining flexible and understanding
+  of unforeseen delays.
+- We will create a supportive space that acknowledges cultural differences by
+  asking and offering help and encouragement from/to teammates facing any challenges
+- We will provide various communication channels whether synchronous or
+  asynchronous to accommodate the different availability times of group members.
+- We will provide constructive criticism while putting into consideration any
+  unforeseen or surprising challenges and refrain from assigning blame.
```
Comment (Bikazemcclent) [inline collaboration/README.md:16]: I don't really understand what's happening

File: collaboration/README.md [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
+- If a breach of trust happens, we will evaluate the mistakes that led to the
+  breach of trust, learn from them, and implement strategies to prevent similar
+  issues in the future.
```
Comment (Bikazemcclent) [inline collaboration/README.md:19]:
```suggestion
  issues in the present.
```

**Review by Bikazemcclent — APPROVED** [review]
looks good!

**Pull request merged**